libs/s25main/controls/ctrlComboBox.cpp

```diff
@@ -176,9 +176,8 @@ bool ctrlComboBox::Msg_WheelDown(const MouseCoords& mc)
 
     if(IsPointInRect(mc.GetPos(), GetDrawRect()))
     {
-        // Scrolled without list opened
-        if(list->GetSelection() + 1 < list->GetNumLines())
-            Msg_ListSelectItem(GetID(), list->GetSelection() + 1);
+        // Will be ignored by the list if to high
+        list->SetSelection(list->GetSelection() + 1);
```

Comment:
perhaps we should provide an "IncSelection" ? (and DecSelection?)

Reply:
I don't think it is required. It is only done like that in the combobox